Makefile

```diff
@@ -0,0 +1,36 @@
+.PHONY = patch train
+
+CVAT_EXPORT_ROOT = 1
```

Comment:
Suggestion: Change this to the project name on CVAT (i.e. Manual)

Reply:
This is just a default value. You can override any Makefile variable, for example:

```
make <target> CVAT_EXPORT_ROOT=Manual
```

Do you mean set the default to `Manual` or something?

Reply:
```suggestion
CVAT_EXPORT_ROOT = "Manual"
```